docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+Service Changes 2018
+====================
+
+As you may have heard, the Open Science Grid (OSG) is undergoing some changes to its fabric of services.
+Some services will be retired, most services will be migrated with little planned effect on sites,
+and some services will be migrated with greater effect on sites.
+This document is intended to guide OSG Site Administrators through these changes.
+The following sections list various OSG services and whether or not site administrator action is required.
+
+!!! note
+    There will be announcements prior to retirement or migration of any OSG service.
+
+The OSG Technology team will be holding [office hours](#office-hours) for any questions or comments about these service 
+changes.
+
+OSG CA
+------
+
+The OSG CA service offers certificate request, renewal, and revocation through the [OIM](#myosg-and-oim) web interface, 
+the OIM REST API, and the `osg-pki-tools`.
+This service will be retired but since the OSG CA will remain in the OSG CA distribution,
```

Comment:
, but because

Reply:
This is a little confusing. We talk about “OSG CA” the service, “OSG CA” the thing (CA), and OSG CA the distribution. Maybe here on line 21, spend a few extra words making this clearer?